web/src/compose_validate.ts

```diff
@@ -746,21 +748,32 @@ export function check_overflow_text(): number {
         set_message_too_long(false);
     } else {
         $indicator.text("");
-        $("textarea#compose-textarea").removeClass("over_limit");
+        $textarea.removeClass("over_limit");
 
         set_message_too_long(false);
     }
 
     return text.length;
+};
+
+export function rewire_check_overflow_text(value: typeof check_overflow_text): void {
```

Comment:
It looks like you aren't using these `rewire_` functions, can you remove these and make the function declarations back to `export function`?

Reply:
There were some [recent work related](https://chat.zulip.org/#narrow/channel/43-automated-testing/topic/override_rewire/near/1977941) to `override_rewire` which requires these functions to be written like above so that we can use `override_rewire`.
Otherwise, we would get something like this:
`You must define rewire_check_overflow_text to use override_rewire on check_overflow_text.`